typescript/widgets/src/components/SearchMenu.tsx

```diff
@@ -157,21 +181,36 @@ export function SearchMenu<
   );
 }
 
-function SearchBar(props: InputProps) {
+const SearchBar = React.forwardRef(function SearchBar(
+  { onChange, value, ...props }: InputProps,
+  ref: React.Ref<HTMLInputElement>,
+) {
   return (
     <div className="htw-relative">
       <SearchIcon
         width={18}
         height={18}
         className="htw-absolute htw-left-4 htw-top-1/2 -htw-translate-y-1/2 htw-opacity-50"
       />
+
       <TextInput
+        onChange={onChange}
+        value={value}
+        ref={ref}
         {...props}
-        className="htw-w-full htw-rounded-lg htw-px-11 htw-py-3"
+        className="htw-bg-transparent focus:htw-bg-transparent htw-border htw-border-gray-200 focus:htw-border-gray-400 htw-w-full htw-rounded-lg htw-px-11 htw-py-3"
```

Comment:
Is the bg-transparent class necessary here? I don't think it's necessary?

Reply:
I have to put `bg-transparent` or `bg-inherit` because the `input` by default comes with a background color, for now I changed to `bg-inherit` so that the color comes directly from the parent.

Lmk if you would rather have `bg-transparent` or if I should change the `input` bg color directly (although this would affect where it is used in other places